cmd/ccli/ccli_test.go

```diff
@@ -15,37 +15,80 @@
 package ccli
 
 import (
+	"context"
+	"crypto/tls"
 	"flag"
+	"os"
 	"strings"
 	"testing"
 
-	log "github.com/golang/glog"
 	"github.com/openconfig/gribigo/compliance"
 	"github.com/openconfig/gribigo/fluent"
 	"github.com/openconfig/gribigo/negtest"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
+
+	spb "github.com/openconfig/gribi/v1/proto/service"
 )
 
 var (
-	addr = flag.String("addr", "", "address of the gRIBI server in the format hostname:port")
+	addr       = flag.String("addr", "", "address of the gRIBI server in the format hostname:port")
+	insecure   = flag.Bool("insecure", false, "dial insecure gRPC (no TLS)")
+	skipverify = flag.Bool("skipverify", true, "allow self-signed TLS certificate")
+	username   = flag.String("username", os.Getenv("USER"), "username to be sent as gRPC metadata")
+	password   = flag.String("password", "", "password to be sent as gRPC metadata")
 )
 
+type flagCred struct{}
+
+func (flagCred) GetRequestMetadata(ctx context.Context, uri ...string) (map[string]string, error) {
+	return map[string]string{
+		"username": *username,
+		"password": *password,
+	}, nil
+}
+
+func (flagCred) RequireTransportSecurity() bool {
```

Comment:
This looks to be unused. I presume it's required to implement the argument for `grpc.WithPerRPCCredentials`? I'd suggest to add a comment to `flagCred` that just states what interface it is implementing.

Reply:
That's right. Added comment to clarify.